### DISCUSSION TITLE
Welcome to xctvplayer Discussions!

### DISCUSSION THREAD

**Discussion by kjbaker-uk** [kjbaker-uk/xctvplayer#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DVergil** [comment]
good luck with the project 🌹

**Reply by kjbaker-uk** [reply]
Thanks I am working on a new version. I started this discussion to help feedback.

**Reply by DVergil** [reply]
i'm testing the non-tv version on my amazon fire stick and it's running great

**Reply by kjbaker-uk** [reply]
Can you upvote my initial thread please as I am trying out a new feature on Github.

**Reply by DVergil** [reply]
Ok

**Reply by HORMIGA-ATOMICA** [reply]
> Buena suerte con el proyecto🌹

Hola podrías ayudarme a quitar los marcos laterales de arriba y abajo no sé cómo se hace te envié un correo te sigo en Twitter enhorabuena por tu proyecto
